### ISSUE TITLE
Installing Smock causes remappings to be changed to Node Modules

### ISSUE THREAD

**Issue by mergd** [wonderland-archive/smock-foundry#27]
First of all –  really neat project. I'm trying this out on a repo that I'm working on right now.

However, I noticed that when I generate remappings using Forge, it changes the remappings.txt file to use use node_modules for some of the non git submodule dependencies instead.

This is what it should be:
```
@openzeppelin/=lib/@openzeppelin/src/
ds-test/=lib/forge-std/lib/ds-test/src/
forge-std/=lib/forge-std/src/
```

This is what it becomes:

```
@defi-wonderland/=node_modules/@defi-wonderland/
@openzeppelin/=node_modules/@openzeppelin/
ds-test/=node_modules/ds-test/
forge-std/=node_modules/forge-std/
solmate/=lib/solmate/src/

```

**Comment by 0xGorilla** [comment]
Hey @mergd , glad you are trying it out! Any chance you have a repository where we can easily reproduce the issue? We will look into it

**Comment by mergd** [comment]
Here's the [link](https://github.com/mergd/femplate/blob/main/GETTING_STARTED.md) to the repo.

My workaround is to change the remapping package script to replace the string

**Comment by gas1cent** [comment]
Hey @mergd! Thanks again for the report and the example repo, it helped a lot. I've just released a new version 1.0.4 that fixes the problem you've encountered and includes a few other improvements. Feel free to try it out. I'd also recommend putting the package in the `devDependencies` section, that way whoever depends on your project won't have to install smock if they don't want to.

**Comment by 0xGorilla** [comment]
Feel free to re-open the issue if it's still valid
